README.md

```diff
@@ -75,6 +75,14 @@ You can also find install via a [zip archive](https://github.com/onivim/oni/rele
 - Double-click on the archive to expand
 - Drag `Oni.app` to the `Applications` folder
 
+To enable key repeat when pressing & holding a key in Oni, write the following in your terminal:
+
+```sh
+defaults write com.extropy.oni ApplePressAndHoldEnabled -bool false
+```
+
+> You may want to add /Applications/Oni.app/Contents/MacOS/ to your $PATH, so that from the console, you can open Oni via `Oni`
```

Comment:
Might not be ideal. The terminal tab is occupied when oni is open, until it closes

Reply:
Oni does have a "Add to Path" command, on MacOS at least, in the Ctrl+Shift+P panel I think. Should probably use that, since it is more easily removable and is done inside Oni.

Reply:
Done